tests/integration/devfile/cmd_devfile_watch_test.go

```diff
@@ -322,6 +195,81 @@ var _ = Describe("odo devfile watch command tests", func() {
 			}, 180, 10).Should(Equal(true))
 
 		})
+
+	})
+
+	When("executing odo watch", func() {
```

Comment:
```suggestion
	When("a component is created and pushed", func() {
```

Also describe what is different from the previous `When("a component is created and pushed")`

Reply:
Updated as suggested